bgpd/bgp_route.c

```diff
@@ -10247,6 +10247,22 @@ bgp_distance_apply (struct prefix *p, struct bgp_info *rinfo, afi_t afi,
     }
 }
 
+static void
+announce_routes_distance_update (struct bgp *bgp)
+{
+  afi_t afi;
+  safi_t safi;
+
+  /* Reannounce all routes to appropriate neighbors */
+  for (afi = AFI_IP; afi < AFI_MAX; afi++)
+      for (safi = SAFI_UNICAST; safi < SAFI_MAX; safi++)
+        {
+            zlog_debug("%s: Announcing routes again"
```

Comment:
Probably also want a space after 'again' or the message will look like
`bgp_announce_routes_distance_update: Announcing routes again(afi=foo, safi=bar)`